go.mod

```diff
@@ -8,12 +8,10 @@ require (
 	github.com/ONSdigital/dp-api-clients-go/v2 v2.2.0
 	github.com/ONSdigital/dp-component-test v0.3.0
 	github.com/ONSdigital/dp-healthcheck v1.1.0
-	github.com/ONSdigital/dp-kafka/v2 v2.2.0
+	github.com/ONSdigital/dp-kafka/v2 v2.4.2
```

Comment:
It is worth updating to use latest go version 1.17 - I would advise updating your go version locally

If you make this change now, it is also worth updating ci files to use the latest go version too

Reply:
go version 1.17.1 updated for mod and ci files